src/settings.ts

```diff
@@ -59,6 +59,7 @@ export const SPARKLEVERSE_LOGO_URL = sparkleverseLogo;
 
 export const DEFAULT_PARTY_NAME = "Anon";
 export const DISPLAY_NAME_MAX_CHAR_COUNT = 40;
+export const DEFAULT_PROFILE_QUESTION_ANSWER = "unknown";
```

Comment:
@sofisparkle is this the correct default answer?

Reply:
Do we need a default answer? I thought we explicitly removed the display of unanswered questions recently in one of OHBM's open source contributions.

Reply:
there was a default variable when I came to that PR, I just reworked that.
I've just checked the logic that was initially here, and realized that it had been changed a bit, in the newer version we did show all the questions even if they hadn't been answered by the user.

So I've fixed that. Now we see only the answered questions (as it was before the PR): https://github.com/sparkletown/sparkle/pull/1499/commits/a5c297f5834a0cb31d6f35d0afe32ddc1f003ac1

(`DEFAULT_PROFILE_QUESTION_ANSWER` is removed)